styles.css

```diff
@@ -0,0 +1,11 @@
+#main{
```

Comment:
You didn't CSS selectors from the brackets and that's why it won't pass the linters test. It would be great if you adhere to that!

Reply:
[OPTIONAL] It sounds more polite. But let's use the magic word 'please' so it will be more polite. Since using the verb at the beginning of the sentence sounds more like a giving order, please use the 'please' word more often 👍  This [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate:~:text=Instead%20of%20giving%20orders%2C%20make%20suggestions.%20%22I%20think%20it%20would%20be%20better%20to%20use%20semantic%20tags%22%20sounds%20nicer%20than%20%22Use%20semantic%20tags%2C%22%20doesn%27t%20it%3F) might be useful to remember how to give a code review. 👨‍💼